docs/metrics/pr-curve.md

```diff
@@ -0,0 +1,108 @@
+---
+search:
+  exclude: true
+---
+
+# Precision-Recall (PR) Curve
+
+!!! info inline end "Guides: Precision and Recall"
+
+    Read the [precision](./precision.md) and the [recall](./recall.md) guides if you're not familiar with those metrics.
+
+A precision-recall (PR) curve is a plot that gauges machine learning model performance by using
+[precision](./precision.md) and [recall](./recall.md), which are performance metrics that evaluate the quality of a
+classification model. The curve is built with precision on the y-axis and recall on the x-axis computed across many
+thresholds, showing a trade-off of how precision and recall values change when a classification threshold changes.
+
+<div class="grid cards" markdown>
+- :kolena-manual-16: API Reference: [`CurvePlot` ↗][kolena.workflow.plot.CurvePlot]
+</div>
+
+## Implementation Details
+
+The curve’s points (precisions and recalls) are calculated with a varying threshold, and made into points (precision
+values on the y-axis and recall values on the x-axis). Precision and recall are threshold-dependent metrics where a
+threshold value must be defined to compute them, and by computing and plotting these two metrics across many thresholds
+we can check how these metrics change depending on the threshold.
+
+??? info "Thresholds Selection"
+
+    Threshold ranges are very customizable. Typically, a uniformly spaced range of values from 0 to 1 can model a PR
+    curve, where users pick the number of thresholds to include. Another common approach to picking thresholds is
+    collecting and sorting the unique confidences of every prediction.
+
+### Example: Binary Classification
+
+Let's consider a simple binary classification example and plot a PR curve at a uniformly spaced range of thresholds.
+The table below shows six samples (four positive and two negative) sorted by their confidence score. Each inference
+is evaluated at each threshold: 0.25, 0.5, and 0.75. It's a negative prediction if its confidence score is below the
+evaluating threshold; otherwise, it's positive.
+
+<center>
+
+| Sample | <nobr>Confidence ↓</nobr> | Inference @ 0.25 | Inference @ 0.5 | Inference @ 0.75 |
+| --- | --- | --- | --- | --- |
+| Positive | 0.9 | <span style="color: green">Positive</span> | <span style="color: green">Positive</span> | <span style="color: green">Positive</span> |
```

Comment:
Couldn't figure out ways to change font color based on the dark/light mode. If anyone has a good idea, let me know!

Reply:
Here's an example (with bad color choices): https://github.com/kolenaIO/kolena/pull/154